### ISSUE TITLE
Show color contrast ratio

### ISSUE THREAD

**Issue by sfhosley** [phallstrom/SafariFontScope#10]
A quick check of the contrast ratio (sort of [like this](https://webaim.org/resources/contrastchecker/)) could be really helpful. Showing if it passes/fails WCAG requirements would also be good. Something like:

`- Contrast: 2.5:1 (AA / AAA Fail)`

or

`- Contrast: 4.5:1 (AA Pass / AAA Fail)`

or

`- Contrast: 7:1 (AA / AAA Pass)`
